### ISSUE TITLE
[MEGA CNC Alpha] Switching between graphs cause OutOFMemory browser error

### ISSUE THREAD

**Issue by kamilzierke** [alex3dbros/megacnc#2]
Switching between "All history" and "Realtime" seems to be creating additional instances of each graph causing an Out Of Memory error for a browser after some time. 
You can replicate the error easily just by clicking between them until UI stops being responsive ant then the tab crashes.

![obraz](https://github.com/alex3dbros/megacnc/assets/67487992/678e08e7-9c20-4d13-a307-6c7ba758e412)

![obraz](https://github.com/alex3dbros/megacnc/assets/67487992/52b3e1d7-a57c-4590-8de4-ac1f09f33794)

Verified on the Edge browser.

**Comment by alex3dbros** [comment]
Yes, I confirm that if switching between graphs can create a weird behaviour, especially when having multiple data points and try to overlay on the graph. I will check it in the future
